src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import Image from "next/image";
+
+import * as S from "./styles";
+
+const ContactUsContent: React.FC = () => {
+  const [name, setName] = useState<string | null>(null);
```

Comment:
What the purpose of using `string | null` here instead of just `string`, what the decision making process here

Reply:
Hem actually I'm not sure lol. Correct me if I'm wrong but it is perfectly legal to do [name, setName] = useState<string>(null). A problem might only arise if name is a more complex data structure with several fields, like described in https://stackoverflow.com/questions/53338922/correct-way-to-type-nullable-state-when-using-reacts-usestate-hook as the type might be inferred to be null. But i think if you alr put string there, there won't be any type confusion.

Reply:
really nice but the question actually is a string vs string | null, not exactly no generic vs string | null but it's fine since you have a very good understanding of it! you even put `useState<string>(null)` actually but I prefer it this way to make it more explicit